gradle/plugins/src/main/kotlin/com.hedera.gradle.versions.gradle.kts

```diff
@@ -24,5 +24,3 @@ plugins {
 group = "com.hedera.hashgraph"
 
 javaPlatform { allowDependencies() }
-
-tasks.register("releaseMavenCentral")
```

Comment:
Why is this registration being removed here?

Reply:
Only need to registry at root level.

Reply:
We found it was being duplicated in com.hedera.gradle.lifecycle.gradle.kts